.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+| Question      | Answer
+| ------------- | ---
+| Tickets       | <!-- URLs to GitHub or JIRA issue(s) (or N/A) -->
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Tests pass?   | yes/no
+| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)
+<!-- Keep in mind: Your contribution has to be compatible with GPL-2.0 as well: https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.html#GPLModuleLicense -->
+
+
+<!-- Replace this comment with Pull Request description -->
+
+
+#### Checklist:
+- [ ] Coding standards (`$ php-cs-fixer fix .`)
+- [ ] Ready for Code Review
```

Comment:
Other thought: let's also add 'Ready for QA'  :stuck_out_tongue: :bowing_man: @micszo @mnocon

Reply:
@alongosz not sure about this. 'Ready for QA' should only by checked after Code Review (especially for Community PR), so this list would reflect a "sort of" workflow (and not a list of things to do before a PR is "ready").

IMHO in an ideal world we should still use 'Ready for QA' label that is set by our developers after they've finished Code Review, I wouldn't use this checklist for that.

Reply:
The idea for TODOs is not to merge PR until they're finished, which is the case for current workflow (unless TODO point says something else). I know from experience that some people look only at TODOs when deciding to merge, so this was the reason for my suggestion.

However you're making a good point that we cannot impose a workflow (review -> QA) here. So now I'm unsure about it myself :)

Reply:
So, the most important question: for whom is this checklist actually?
* If it's for the external contributor (or our developer), then it should simply remind him about things he should do (like check CS, write good PR description, tidy up commits etc.) before he can create a PR that will be worth looking at.
* If it's for the maintainer too then points about Code Review/QA can be added (but it then becomes a workflow, and we have already JIRA/labels for that). 

I think it should be option 1 :)

Reply:
For me, it's just a checklist of things to do before considering pull request as "Ready". The last option is just another way of saying "OK, it's no longer WIP, I consider it done".

It should not be a workflow.

Reply:
Ok, so dropping my suggestion :)